feat: create using discover services with JS page (#1374)
agrc/gis.utah.gov: _posts/2015-12-21-using-agrcs-new-web-mercator-services-in-your-web-maps.md

  - Developer
  - Featured
---

> <p class="pop text-center">
>   <strong>Note:</strong> Please go to <a href="{% link developer/base-maps/discover/index.md %}">this page</a> for more updated information on using Discover services with the Esri JavaScript API v4.x.
> </p>
> 

Reviewer: remove click here type link

```suggestion
<p class="pop text-center">
  The instructions on this page are dated. <a href="{% link developer/base-maps/discover/index.md %}">Using Discover services with the Esri JavaScript API v4.x</a> contains more current information.
</p>

```
